parent/pom.xml

```diff
@@ -1458,6 +1458,10 @@
                            <key>infinispan.isCoreModule</key>
                            <value>${module.metadata.isCoreModule}</value>
                         </systemProperty>
+                        <systemProperty>
+                           <key>log4j.configurationFile</key>
+                           <value>${log4j.configurationFile}</value>
+                        </systemProperty>
```

Comment:
That is actually not needed anymore, since I've removed log4j from the classpath